package.json

```diff
@@ -18,6 +18,18 @@
       "title": "Works for Private Browsing",
       "type": "bool",
       "value": true
+    },
+    {
+      "name": "maxOpenInNextWindow",
+      "title": "Open in next available window",
```

Comment:
"Open in next available window" looks fine for me, because the word "available" already implies that the current one reached the maximum number of tabs and it's unavailable.

You can of course be more explicit if you want to, but use "exceeded maximum" sounds weird, since the whole point of the addon is never "to exceed" a specific number of tabs.

My two cents! ;-)